test/xbgpu/test_engine.py

```diff
@@ -389,6 +400,9 @@ def verify_beam_sensors(
         The beam weights, quantiser-gains and delays applied to each input of
         the beam data product. These are real floating-point values generated
         for the unit test.
+    saturation
+        Expected per-beam saturationcounts. These are approximate because the
```

Comment:
Aha, this is outdated indeed. HEAD shows
> Lower and upper bounds on per-beam saturation counts. A range is
necessary because dithering is not modelled on the host.